pkg/nodegraph/Library/RapsaiNodes/RapsaiImagesNode.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+
+export const RapsaiImagesNode = {
```

Comment:
do we want to give these a more generic name?

Reply:
Not sure which bit you mean to genericize. 

I'm very specifically trying to suggest these are "tools offered by the Rapsai project," so that they can position themselves as vendors of high-quality Nodes (if they like).